.github/workflows/test.yml

```
@@ -17,68 +17,92 @@ concurrency:
       - uses: ./.github/actions/setup-all
       - uses: ./.github/actions/setup-test
 
-      - name: Start SSH Agent
-        uses: webfactory/ssh-agent@v0.9.0
-        with:
-          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY_RO }}
+      - name: Start SSH Agent & Add Key
+        shell: bash -l {0}
+        run: |
+          mkdir -p ~/.ssh
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)